### PR TITLE
Add About/contact section

### PR DESCRIPTION
**Pull request by abregorivas**
The following changes were made:
- Create about & contact includes
- Create collection files for contact & about
- Configured collection and connect to index.html

### CODE REVIEW

File: _contact_us/contact-us.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+w---
```
Comment (thekaveman) [inline _contact_us/contact-us.md]: this was probably from a vim `:wq` right?

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: _contact_us/contact-us.md (hunk 1 above)
Comment (abregorivas) [reply _contact_us/contact-us.md]: lol! yes it was trying to exit

**Review by thekaveman — CHANGES_REQUESTED** [review]
This is almost there. I think to get the site up:

1. convert to simple includes like you did for the hero
2. rebase on the latest `gh-pages` (and careful with those whitespace changes on e.g. index)

**Comment by abregorivas** [comment]
Rebased and made all the changes

**Review by thekaveman — APPROVED** [review]

**Pull request merged**